src/Application/Command/BumpChangelogForReleaseBranch.php

```diff
@@ -44,12 +44,15 @@ public function execute(InputInterface $input, OutputInterface $output): int
     {
         $milestoneClosedEvent = ($this->loadEvent)();
         $repositoryName       = $milestoneClosedEvent->repository();
-        $repositoryCloneUri   = $repositoryName->uriWithTokenAuthentication($this->environment->githubToken());
         $repositoryPath       = $this->environment->githubWorkspacePath();
 
         Psl\invariant(Filesystem\is_directory($repositoryPath . '/.git'), 'Workspace is not a GIT repository.');
 
-        ($this->fetch)($repositoryCloneUri, $repositoryPath);
+        ($this->fetch)(
+            $repositoryName->uri(),
+            $repositoryName->uriWithTokenAuthentication($this->environment->githubToken()),
+            $repositoryPath
+        );
```

Comment:
Trying to understand how this fixes the problem - is it because in the case of a crash, only the first argument (here, `$repositoryName->uri()`, previously, a version of the URI with the token included) is ever logged?

Reply:
It was in the exception message.

I think a validating test case is needed.